cea/technologies/cogeneration.py

```diff
@@ -141,14 +141,14 @@ def CC_Op(wdot_W, gt_size_W, fuel, tDH_K, gV) :
 
     """
 
-    (eta0, mdot0_kgpers) = GT_fullLoadParam(gt_size_W, fuel, gV)
-    (eta, mdot_kgpers, texh_K, mdotfuel_kgpers) = GT_partLoadParam(wdot_W, gt_size_W, eta0, mdot0_kgpers, fuel, gV)
-    (qdot_W, wdotfin_W) = ST_Op(mdot_kgpers, texh_K, tDH_K, fuel, gV)
+    (eta0, mdot0_kgpers) = GT_fullLoadParam(gt_size_W, fuel, gV, optimization_constants)
+    (eta, mdot_kgpers, texh_K, mdotfuel_kgpers) = GT_partLoadParam(wdot_W, gt_size_W, eta0, mdot0_kgpers, fuel, gV, optimization_constants)
```

Comment:
I see the variable gV still being passed one. should not it have been completely remove with this PR?